index.html

```diff
@@ -149,9 +149,16 @@ <h1>Basic usage</h1>
 	<pre><code>&lt;pre>&lt;code class="language-css">p { color: red }&lt;/code>&lt;/pre></code></pre>
 	<p>If you use that pattern, the <code>&lt;pre></code> will automatically get the <code>language-xxxx</code> class (if it doesn’t already have it) and will be styled as a code block.</p>
 
-	<p>If you want to prevent any elements from being automatically highlighted, you can use the attribute <code>data-manual</code> on the <code>&lt;script></code> element you used for prism and use the <a href="extending.html#api">API</a>.
+	<p>If you want to prevent any elements from being automatically highlighted and instead use the <a href="extending.html#api">API</a>, you can set <code>Prism.manual = true;</code> before <code>DOMContentLoaded</code> is fired. By setting the attribute <code>data-manual</code> on the <code>&lt;script></code> element you used for prism this will be done automatically.
```

Comment:
> before <code>DOMContentLoaded</code> is fired

Not only that but also before the timeouts we use otherwise. I wouldn't mention any specific DOM events.

Reply:
`DOMContentLoaded` is the latest point you can _rely_ on. If the page is already loaded and Prism is dynamically embedded one is not able to reliably set manual (unless the file containing Prism also sets the flag), because you cannot ensure that your code runs before the timers run.

So while technically it's not 100% precise I believe the information is valuable for consumers that cannot simply set `data-manual`.

Reply:
> one is not able to reliably set manual

> However I prefer the current version, because your suggestion will break if e.g. async is put onto the script tag or Prism is loaded by a loader such as require.js or head.js.

As you implied, it's possible to reliably set `manual` using the example you added to the site.

That aside, You convinced me that we should mention the DOM event.

Also, here is a small suggestion to make the doc a bit more specific:

```diff
-, you can set <code>Prism.manual = true;</code> before <code>DOMContentLoaded</code> is fired.
+, you can set <code>Prism.manual</code> to <code>true</code> before the <code>DOMContentLoaded</code> event is fired.
```

And

```diff
-By setting the attribute <code>data-manual</code> on the <code>&lt;script></code> element you used for prism this will be done automatically.
+By setting the <code>data-manual</code> attribute on the <code>&lt;script></code> element containing Prism core, this will be done automatically.
```

Reply:
Changed to your suggestion.